rollup-common.js

```diff
@@ -205,30 +210,71 @@ export function litRollupConfig({
         // way or another, so it's difficult to define a default in the source code
         // itself.
         replace({
-          "const DEV_MODE = true": "const DEV_MODE = false",
+          'const DEV_MODE = true': 'const DEV_MODE = false',
         }),
         // This plugin automatically composes the existing TypeScript -> raw JS
         // sourcemap with the raw JS -> minified JS one that we're generating here.
         sourcemaps(),
         terser(terserOptions),
         summary(),
-        ...(CHECKSIZE ? [skipBundleOutput] : [
-          // Place a copy of each d.ts file adjacent to its minified module.
-          copy({
-            targets: entryPoints.map((name) => ({
-              src: `development/${name}.d.ts`,
-              dest: pathLib.dirname(name),
-            })),
-          }),
-          // Copy platform support tests.
-          copy({
-            targets: [{
-              src: `src/test/platform-support/*_test.html`,
-              dest: ['development/test/platform-support', 'test/platform-support'],
-            }]
-          }),
-        ]),
-      ]
+        ...(CHECKSIZE
+          ? [skipBundleOutput]
+          : [
+              // Place a copy of each d.ts file adjacent to its minified module.
+              copy({
+                targets: entryPoints.map((name) => ({
+                  src: `development/${name}.d.ts`,
+                  dest: pathLib.dirname(name),
+                })),
+              }),
+              // Copy platform support tests.
+              copy({
+                targets: [
+                  {
+                    src: `src/test/platform-support/*_test.html`,
+                    dest: [
+                      'development/test/platform-support',
+                      'test/platform-support',
+                    ],
+                  },
+                ],
+              }),
+            ]),
+      ],
     },
+    ...bundled.map(({file, output, name}) =>
+      litBundle({
+        file,
+        output,
+        name,
+        terserOptions: terserOptions,
+      })
+    ),
   ];
-}
\ No newline at end of file
+}
+
+export const litBundle = ({
```

Comment:
`litBundle` and `litRollupConfig` are kinda undifferentiated now. How about something like:

- `litRollupConfig` -> `litProdConfig`
- `litBundle` -> `litMonoBundleConfig`

(Probably this whole rollup config organization needs a re-think now, like inverting the control somewhat by exporting some smaller pieces instead of these big functions, but we can think about that another time. Maybe I'll take a pass at it someday).

Reply:
Done. And happy to have you clean this up as a follow up.